src/Script/RamCalculations.ts

```diff
@@ -49,6 +49,7 @@ async function parseOnlyRamCalculate(
   otherScripts: Script[],
   code: string,
   workerScript: WorkerScript,
+  scriptPath: string,
```

Comment:
Parameter needs to be added to the doc comment. Should indicate that it is the full path to the script that is currently in the code parameter.